### PR TITLE
partitioning from delta handle

### PR DESCRIPTION
**Pull request by mrmasterplan**
Get partitioning from delta handle in a nice format.

### CODE REVIEW

File: src/atc/delta/delta_handle.py [hunk 1]
```diff
@@ -110,6 +112,35 @@ def recreate_hive_table(self):
...
+        +-----------------+---------------+-------+
+        |         col_name|      data_type|comment|
+        +-----------------+---------------+-------+
+        |           mycolA|         string|       |
+        |           myColB|            int|       |
+        |                 |               |       |
+        |   # Partitioning|               |       |
+        |           Part 0|         mycolA|       |
+        +-----------------+---------------+-------+
+        but thie method return the partitioning in the form ['mycolA']
```
Comment (LauJohansson) [inline src/atc/delta/delta_handle.py:126]: misspelling

File: src/atc/delta/delta_handle.py [hunk 2]
```diff
@@ -110,6 +112,35 @@ def recreate_hive_table(self):
...
+        """
+        if self._partitioning is None:
+            rows_iter = iter(
+                Spark.get().sql(f"DESCRIBE TABLE {self.get_tablename()}").collect()
+            )
+            for row in rows_iter:
```
Comment (LauJohansson) [inline src/atc/delta/delta_handle.py:132]: could you provide a # comment on what this logic does?

File: src/atc/delta/delta_handle.py [hunk 3]
```diff
@@ -110,6 +112,35 @@ def recreate_hive_table(self):
         self.drop()
         self.create_hive_table()
 
+    def get_partitioning(self):
```
Comment (LauJohansson) [inline src/atc/delta/delta_handle.py:115]: what if there is no partion? Is that even possible?

Comment (LauJohansson) [reply src/atc/delta/delta_handle.py:115]: if it is - how does the method handle it?

**Review by LauJohansson — DISMISSED** [review]

File: src/atc/delta/delta_handle.py (hunk 3 above)
Comment (mrmasterplan) [reply src/atc/delta/delta_handle.py:115]: its possible, returns empty list. I added a test.

File: src/atc/delta/delta_handle.py (hunk 2 above)
Comment (mrmasterplan) [reply src/atc/delta/delta_handle.py:136]: added comments.

File: src/atc/delta/delta_handle.py (hunk 1 above)
Comment (mrmasterplan) [reply src/atc/delta/delta_handle.py]: fixed

**Review by LauJohansson — APPROVED** [review]

**Review by Okami1 — COMMENTED** [review]
Ser godt ud, ingen kommentarer for ændringer

**Pull request merged**